Yoast/wordpress-seo
# Register Gutenberg sidebar

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Put svg icon in images folder.
* Kept the entire registration and creation inside one function for now, I suspect this can be done better, but i'm not sure right now how, so pushing it along for now.

## Test instructions

This PR can be tested by following these steps:

* check if all unit-tests pass.
* Open the gutenberg editor, and check if Yoast SEO has been added as a menu item with it's icon. Clicking on it should open a sidebar area with some temporary content.

## Quality assurance


Fixes #9509
